config/routes.rb

```diff
@@ -7,6 +7,8 @@
   root 'welcome#index'
   get 'game_day', to: 'game_day#index', as: 'game_day'
   resources :seasons, :episodes, :contestants, :drafts, :tribes
+  resources :users, except: :show
+  get 'users/:id/profile', to: 'users#show', as: 'user_path'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.